src/main/java/com/amazonaws/kinesisvideo/producer/KvsRetryStrategy.java

```diff
@@ -0,0 +1,27 @@
+package com.amazonaws.kinesisvideo.producer;
+
+public class KvsRetryStrategy {
+
+    private final long mRetryStrategy;
+    private final long mRetryStrategyConfig;
+    private final KvsRetryStrategyType mKvsRetryStrategyType;
+
+    public KvsRetryStrategy() {
+        mRetryStrategy = 0;
+        mRetryStrategyConfig = 0;
+        mKvsRetryStrategyType = KvsRetryStrategyType.DISABLED;
```

Comment:
Confirm if we want the setting to be DISABLED. We would still need exponential retry, just that PIC should use the default functions. Let us understand what happens if the value is disabled. Does PIC override it or actually disables exponential backoff